packages/framework/src/Model/PriceList/ImportPriceListResult.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Model\PriceList;
+
+class ImportPriceListResult
```

Comment:
Could you create a factory for this class, please?

Reply:
sure, a8684ce223